### PR TITLE
Introduce new theme for WaitForTransaction component

### PR DESCRIPTION
**Pull request by JustCaptcha**

### CODE REVIEW

File: universal-login-react/src/ui/commons/CompanyLogo.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+// import UImage from '../../assets/U.svg';
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/commons/CompanyLogo.tsx]: remove comment

File: universal-login-react/src/ui/commons/Notice.tsx [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
 export interface NoticeProps {
   message?: string;
 }
 
 export const Notice = ({message}: NoticeProps) => {
+  const theme = useThemeClassFor();
   if (message) {
     return (
-      <div className={'notice-wrapper'}>
+      <div className={`${theme} notice-wrapper`}>
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/commons/Notice.tsx]: `useThemeClass()`

Comment (JustCaptcha) [reply universal-login-react/src/ui/commons/Notice.tsx]: "React Hook "useThemeClassFor" is called conditionally. React Hooks must be called in the exact same order in every component render" That's why I use
`const theme = useThemeClassFor();`

File: universal-login-react/src/ui/commons/WaitingForTransaction.tsx [hunk 3]
```diff
@@ -3,38 +3,47 @@ import {PublicRelayerConfig} from '@universal-login/commons';
...
 );
 
 export const WaitingForTransaction = (props: WaitingForTransactionProps) => (
-  <div className={`unilogin-component-waitingfortransaction ${useThemeClassFor()}`}>
+  <div className={useClassFor('waitingfortransaction')}>
+    <CompanyLogo />
+    <ButtonClose onClick={() => console.log('hide modal')}/>
     <div className={getStyleForTopLevelComponent(props.className)}>
-      <Body {...props} />
+      <Body {...props}
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/commons/WaitingForTransaction.tsx]:
```ts
const WAITING_FOR_TRANSACTION_DESCRIPTION = 'It takes time to register your username and deploy your wallet. In order to do so, we need to create a transaction and wait until the Ethereum blockchain validates it.';

<Body description = WAITING_FOR_TRANSACTION_DESCRIPTION {..props} />
```

File: universal-login-react/src/ui/commons/WaitingForTransaction.tsx [hunk 4]
```diff
@@ -3,38 +3,47 @@ import {PublicRelayerConfig} from '@universal-login/commons';
...
     </div>
     {info && <p className={classForComponent('info-text')}>{info}</p>}
   </div>
 );
 
 export const WaitingForTransaction = (props: WaitingForTransactionProps) => (
-  <div className={`unilogin-component-waitingfortransaction ${useThemeClassFor()}`}>
+  <div className={useClassFor('waitingfortransaction')}>
+    <CompanyLogo />
+    <ButtonClose onClick={() => console.log('hide modal')}/>
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/commons/WaitingForTransaction.tsx]: Style button in ModalWrapper. Remove the new one.

File: universal-login-react/src/ui/commons/Button/ButtonClose.tsx [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {useClassFor} from '../../utils/classFor';
+import '../../styles/base/buttonClose.sass';
+
+interface ButtonCloseProps {
+  onClick: Function;
+};
+
+export const ButtonClose = ({onClick}: ButtonCloseProps) => {
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/commons/Button/ButtonClose.tsx]: We can remove it

File: universal-login-react/src/ui/commons/Notice.tsx [hunk 6]
```diff
@@ -1,15 +1,17 @@
 import React from 'react';
+import {useThemeClassFor, classForComponent} from '../utils/classFor';
 import './../styles/notice.sass';
-
+import './../styles/themes/Jarvis/noticeThemeJarvis.sass';
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/commons/Notice.tsx]: missing empty line

File: universal-login-react/src/ui/styles/themes/Jarvis/noticeThemeJarvis.sass [hunk 7]
```diff
@@ -0,0 +1,29 @@
+.unilogin-theme-jarvis.unilogin-component-notice-wrapper
+  // display: none
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/styles/themes/Jarvis/noticeThemeJarvis.sass]: comment

File: universal-login-react/src/ui/styles/themes/Legacy/waitingForTransactionThemeLegacy.sass [hunk 8]
```diff
@@ -17,3 +17,12 @@
   font-size: 16px
   color: #31405E
   line-height: 24px
+
+.unilogin-theme-default .unilogin-component-waitingfortransaction-txn-img
+  display: none
+
+.unilogin-theme-default .unilogin-component-button-close
+  display: none
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/styles/themes/Legacy/waitingForTransactionThemeLegacy.sass]: We can remove these 2 lines

**Pull request merged**